### PR TITLE
feat: LT - 1593 - F680 Clearance Types

### PR DESCRIPTION
**Pull request by andytaylorkainos**

### CODE REVIEW

File: applications/views/f680_details.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+class F680Details(SingleFormView):
+    def init(self, request, **kwargs):
+        self.object_pk = str(kwargs["pk"])
+        application = get_application(request, self.object_pk)
+        application["f680_clearance_types"] = [
+            f680_clearance_type["name"]["key"] for f680_clearance_type in application["f680_clearance_types"]
+        ]
+        self.form = f680_details_form(request, self.object_pk)
+        self.action = put_application_with_clearance_types
+        self.data = {"f680_clearance_types": application["f680_clearance_types"]}
```
Comment (jfaracik) [inline applications/views/f680_details.py]: Could line 17 not be:

```
self.data = {"f680_clearance_types": f680_clearance_type["name"]["key"] for f680_clearance_type in application["f680_clearance_types"]}
```

File: ui_automation_tests/pages/apply_for_a_licence_page.py [hunk 2]
```diff
@@ -41,6 +42,12 @@ def select_goods_categories(self):
         for checkbox in checkboxes:
             checkbox.click()
 
+    def select_types_of_clearance(self):
+
```
Comment (jfaracik) [inline ui_automation_tests/pages/apply_for_a_licence_page.py]: Superfluous space

File: ui_automation_tests/step_defs/test_submit_mod_application.py [hunk 3]
```diff
@@ -10,3 +10,14 @@
 def create_mod_application(driver, context, type):  # noqa
     ApplyForALicencePage(driver).select_mod_application_type(type)
     functions.click_submit(driver)
+
+
+@when("I choose types of clearance I need")
```
Comment (jfaracik) [inline ui_automation_tests/step_defs/test_submit_mod_application.py]: I choose _the_ types of clearance I need

File: applications/views/f680_details.py (hunk 1 above)
Comment (andytaylorkainos) [reply applications/views/f680_details.py]: yep!

**Pull request merged**